packages/calico/extra/dcos-calico-node.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=DC/OS calico node: initializes a new calico node and runs calico components
+After=docker.service
+Requires=docker.service
+ConditionPathExists=/opt/mesosphere/etc/calico_enabled
+
+[Service]
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/calico-node.env
+ExecStartPre=/usr/bin/mkdir -p /etc/calico
+ExecStartPre=/usr/bin/cp /opt/mesosphere/etc/calico/calicoctl.cfg /etc/calico/calicoctl.cfg
+ExecStartPre=-/usr/bin/docker rm -f dcos-calico-node
```

Comment:
How about moving all these steps in `dcos-calico-node.sh` itself so that we need just one place to worry about?

Reply:
Sure, thanks.